Assets/MixedRealityToolkit/Inspectors/Profiles/BaseMixedRealityToolkitConfigurationProfileInspector.cs

```diff
@@ -232,6 +234,22 @@ protected void RenderProfileHeader(string title, string description, Object sele
             EditorGUILayout.LabelField(string.Empty, GUI.skin.horizontalSlider);
         }
 
+        /// <summary>
+        /// If application is playing, then show warning to the user and disable inspector GUI
+        /// </summary>
+        /// <returns>true if application is playing, false otherwise</returns>
+        protected bool CheckEditorPlayMode()
+        {
+            if (Application.isPlaying)
+            {
+                EditorGUILayout.HelpBox("Mixed Reality Toolkit settings cannot be edited while in play mode.", MessageType.Warning);
+                GUI.enabled = false;
```

Comment:
I don't want to over-complicate this, but have you considered using [DisabledScope](https://docs.unity3d.com/ScriptReference/EditorGUI.BeginDisabledGroup.html) instead of `GUI.enabled`? `GUI.enabled` can sometimes have ugly side effects.

Reply:
Interesting, didn't even know that was a thing. The only thing I'll have to test is if DisableScope also sets GUI.enabled. Because all of the sub-profiles read GUI.enabled if it was already disabled. So lots of code may break if I try to disable via this was but GUI.enabled is still true?

Reply:
Yeah, it may not be worth holding up this PR. My impulse is to merge this as-is and clean up GUI.enabled all in one go in a separate PR. But I'll defer to @wiwei and @davidkline-ms.

Reply:
Sounds good to me.

Reply:
Great, agreed. Leave that investigation to switch everything over for disablegroup to another PR